modules/claims_api/app/swagger/claims_api/v2/production/swagger.json

```diff
@@ -2409,6 +2410,10 @@
                               "type": "boolean",
                               "description": "Claimant certifies and authorizes release of information.",
                               "default": false
+                            },
+                            "claimId": {
+                              "type": "string",
+                              "example": "600517517"
```

Comment:
Not Relevant to this PR: this is interesting. did we not have this in yet?

Reply:
oh wait, this shouldn't be in the root /526.. this should only be in synchronous.. is this something caused by the Flip Flop Rockwell was resolving?

Reply:
@FonzMP  That flip/flop was a mixup with the description text for  the sync and async endpoints so if there is an issue here that was not something I was addressing.

Reply:
@stiehlrod I will need to defer to Alex on this one, I thought that was included in the response for the sync endpoint so should be good (?), although I'm not sure why it is getting added in here as something new